credentials.json

```diff
@@ -0,0 +1,11 @@
+{
+  "web": {
+    "client_id": "575440888732-lst2joh94qfn4tcv7ae17sbos4cbikk9.apps.googleusercontent.com",
+    "project_id": "hubbabubba-kidda-1537413526525",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://www.googleapis.com/oauth2/v3/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_secret": "GKETWr7tnUWBXIIXwN8dTyDM",
```

Comment:
Do we want to be uploading this stuff to the repo? This seems like it would be bad to have openly available on github.